lib/object.c

```diff
@@ -9,7 +9,6 @@
 #include "log.h"
 #include "object.h"
 #include "tpm2_auth_util.h"
-#include "tpm2_hierarchy.h"
```

Comment:
dont you need this?

Reply:
Moved it to object.h. Required when defining tpm2_util_object_load_auth.